<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CasePrices</class>
 <widget class="QDialog" name="CasePrices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #232323;
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>Змеиный укус:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>Кейс Грёзы и Кошмары:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>Призма 2 Кейс:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>Кейс Разлом:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>Гремучий кейс:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>Кейс Решающий момент:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 40px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>    Кейс</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 40px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>    Цена</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: gold;
	width: 75px;
	height: 50px;
	font-size: 14px;
	font-weight: bold;
	border: none;
	text-align: center;
}

QPushButton:hover {
	background-color: orange;
}

QPushButton:pressed {
	background-color: yellow;
}</string>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
